xt/aspell.t

```diff
@@ -24,7 +27,7 @@ text that is part of a code example)
 
 my @files = Test-Files.documents.grep({not $_ ~~ / 'README.' .. '.md' /});
 
-plan +@files;
+plan +@files *2;
```

Comment:
A tiny nitpick, but it should be either `+@files * 2` or `+@files*2`?

Reply:
f6d325843